examples/sites/src/views/components/components.vue

```
@@ -1,763 +1,443 @@
+import ComponentHeader from './header.vue'
+import ComponentContributor from './contributor.vue'
+import ApiDocs from './api-docs.vue'
+import useTasksFinish from './composition/useTasksFinish'
+
+defineOptions({
+  name: 'CmpPageVue'
+})
 
-    const { apiModeState } = useApiMode()
-    const { templateModeState, staticPath, optionsList } = useTemplateMode()
+const props = defineProps({ loadData: {} })
```

ESLint

[error] defineProps should be the first statement in `<script setup>` (after any potential import statements or type definitions).

(vue/define-macros-order)